storage_service/locations/models/pipeline.py

```diff
@@ -126,3 +128,76 @@ def create_default_locations(self, shared_path=None):
                     p['purpose'], location, self)
                 LocationPipeline.objects.get_or_create(
                     pipeline=self, location=location)
+
+    # HTTP API CALLS
+
+    def _request_api(self, method, path, **kwargs):
+        url = 'http://{}/api/{}'.format(
+            self.remote_name,
+            path.lstrip('/'),
+        )
+        fields = {
+            'username': self.api_username,
+            'api_key': self.api_key
+        }
+        if 'fields' in kwargs:
```

Comment:
If this is the only thing you're using `kwargs` for, maybe make it a defined parameter with a default of `None`?

```python
def _request_api(self, method, path, fields=None)
# ...

if fields is not None:
    fields.update(kwargs['fields'])
```